### PR TITLE
Add implementation of standard deviation on data

### PR DESCRIPTION
**Pull request by bennetjos**
Write something....

### CODE REVIEW

File: inflammation-analysis.py [hunk 1]
```diff
@@ -27,7 +27,8 @@ def main(args):
     for filename in InFiles:
         inflammation_data = models.load_csv(filename)
 
-        view_data = {'average': models.daily_mean(inflammation_data), 'max': models.daily_max(inflammation_data), 'min': models.daily_min(inflammation_data)}
+        view_data = {'average': models.daily_mean(inflammation_data), 'max': models.daily_max(inflammation_data), 'min': models.daily_min(inflammation_data), **(models.s_dev(inflammation_data))}
```
Comment (Samuel-Bakes) [inline inflammation-analysis.py]: Needs style updates, consider multi line

File: inflammation/models.py [hunk 2]
```diff
@@ -54,3 +54,13 @@ def daily_min(data):
     """Calculate the daily min of a 2d inflammation data array."""
     return np.min(data, axis=0)
 
+
```
Comment (Samuel-Bakes) [inline inflammation/models.py:72]: Variable names unclear for mmm and devs. 
return variable is called s_dev2 - is the function returning standard deviation squared? Description says it returns the standard deviation

**Review by Samuel-Bakes — CHANGES_REQUESTED** [review]
good, function computes standard deviation and plots results as required by SR1.1.1. Needs some revision on variable names and the following test cases: 

1. Does the plotting work as expected
2. Does the function work for negative numbers 
3. Different array sizes (one patient, multiple days. multiple patients, one day.)

File: inflammation-analysis.py (hunk 1 above)
Comment (bennetjos) [reply inflammation-analysis.py]: Changed Accordingly

File: inflammation/models.py (hunk 2 above)
Comment (bennetjos) [reply inflammation/models.py:72]: Changed accordingly

**Review by Samuel-Bakes — APPROVED** [review]

**Pull request merged**